.travis.yml

```diff
@@ -84,6 +88,7 @@ script:
   ./tools/build.py -C target/release -j2
   DENO_BUILD_MODE=release ./tools/test.py
 
+- sccache --stop-server
```

Comment:
I don't think this is necessary

Reply:
Not necessary but I want to see the stats.